_plugins/README.md

```diff
@@ -3,6 +3,9 @@
 This directory contains `*.rb` files that extend the original Jekyll
 classes and provide custom formatting for the docs site:
 
+Notes:
+N/A: Not Available/ Not applicable
```

Comment:
please change the content where we use N/A to more relevant text rather than adding a note here